Documentation for Azure Application Code Assessment Tool (#37523)
dotnet/docs: docs/azure/migration/appcat/install.md

---
title: Azure AppCAT Installation
description: Learn how to install the Azure Application and Code Assessment Toolkit.
ms.topic: conceptual
ms.date: 10/16/2023
author: codemillmatt
ms.author: masoucou
---

# Install the Azure Application and Code Assessment Toolkit (Azure AppCAT)

> Azure AppCAT for .NET can be installed as a Visual Studio extension of a s a.NET command-line tool. When installed as a Visual Studio extension, loaded projects can be upgraded through the context menu. The command-line version of the tool provides and interactive step-by-step experience.

Reviewer: ```suggestion
Azure AppCAT for .NET can be installed as a Visual Studio extension or as a .NET command-line tool. When installed as a Visual Studio extension, loaded projects and solution can be upgraded through the context menu in Solution Explorer. The command-line version of the tool provides and interactive step-by-step experience.
```

---
title: Azure Migrate application and code assessment for .NET installation
description: Learn how to install Azure Migrate application and code assessment for .NET
ms.topic: conceptual
ms.date: 11/09/2023
author: codemillmatt
ms.author: masoucou
---

# Install Azure Migrate application and code assessment for .NET

> Application and code assessment can be installed as a Visual Studio extension or as a .NET command-line tool. When installed as a Visual Studio extension, loaded projects can be upgraded through the context menu in Solution Explorer. The command-line version of the tool provides an interactive step-by-step experience.

Reviewer: ```suggestion
Application and code assessment can be installed as a Visual Studio extension or as a .NET command-line tool. When installed as a Visual Studio extension, loaded projects can be analyzed through the context menu in Solution Explorer. The command-line version of the tool provides an interactive step-by-step experience.
```

## Prerequisites

- Windows operating system

> - Visual Studio 2022 version 17.1 or later

Reviewer: ```suggestion
- Visual Studio 2022 version 17.1 or later - for Visual Studio extension
```

> - .NET SDK for the command-line tool

Reviewer: ```suggestion
```

> - .NET SDK for the command-line tool

Reviewer: ```suggestion
- .NET SDK - for the command-line tool
```

## Install the Visual Studio extension

> The application and code assessment for .NET can be installed as a Visual Studio extension, which lets you analyze an open projects in your solution. Use the following steps to install it from inside Visual Studio. Alternatively, you can download and install the extension from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.azureappcat).

Reviewer: ```suggestion
The application and code assessment for .NET can be installed as a Visual Studio extension, which lets you analyze an open projects in your solution. Use the following steps to install it from inside Visual Studio. Alternatively, you can download and install the extension from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.appcat).
```

  1. With Visual Studio opened, press the **Extensions > Manage Extensions** menu item, which opens the **Manage Extensions** window.
  2. In the **Manage Extensions** window, enter **"Azure Migrate"** into the search input box.
  3. Select the **Azure Migrate application and code assessment** item, and then select **Download**.

>   4. Once the extension has been downloaded, close Visual Studio. This starts the the installation of the extension.

Reviewer: ```suggestion
  4. Once the extension has been downloaded, close Visual Studio. This starts the installation of the extension.
```

>   5. Select **Modify** and follow the directions to install the extension.

Reviewer: ```suggestion
  5. In the VSIX Installer dialog elect **Modify** and follow the directions to install the extension.
```

>   5. In the VSIX Installer dialog elect **Modify** and follow the directions to install the extension.

Reviewer: ```suggestion
  5. In the VSIX Installer dialog select **Modify** and follow the directions to install the extension.
```

## Install the .NET global tool

Azure Migrate application and code assessment for .NET is also available as a .NET global tool. You can install the tool with the following command.

```dotnetcli
dotnet tool install -g dotnet-appcat
```

> Similarly, because the AppCAT tool is installed as a .NET tool, it can be easily updated by running:

Reviewer: ```suggestion
Similarly, to update the tool, use the following command:
```
